prisma/schema.prisma

```diff
@@ -7,7 +7,8 @@ generator client {
 }
 
 datasource db {
-    provider = "sqlite"
+    provider = "mysql"
```

Comment:
so now you need to set up mysql DB locally?

Reply:
This is still a WIP because I'm not sure if I want to implement this or not but this is just schema change for it to work for a hosted DB. When you visit the hosted project URL (https://bread-meta.vercel.app/), authentication was an issue on both Google and GitHub. Now, these changes make it so you are allowed to authenticate with Google but still working on GitHub. This change also works locally so there is no effect on productivity.